Updates to documentation

maxmatthews opened the pull request:
Updates to the sample file and documentation so they are in sync. The port number also was in the wrong place. 
https://github.com/zodern/meteor-up/issues/355

arunoda commented:
Okay. This seems legit to me.

arunoda commented:
Did a release with `1.4.1`

chmac commented:
Awesome, thanks for the incredibly fast response. I didn't even have to write up instructions for my colleagues, it's already fixed! :+1: :-)

maxmatthews committed:
Move the port setting in readme

The port location in the readme is incorrect. It belongs in the env settings and must be capitalized.

maxmatthews committed:
Change the format of docker image to match readme

The code handles the old format, but this is the format that is documented in the readme.

maxmatthews committed:
Update section in the readme regarding port #

zodern commented on README.md:
```diff
@@ -287,12 +287,16 @@ We need to have two separate Meteor Up projects. For that, create two directorie
 In the staging `mup.js`, add a field called `appName` with the value `staging`. You can add any name you prefer instead of `staging`. Since we are running our staging app on port 8000, add an environment variable called `PORT` with the value 8000.
 
-You might also have to tell docker to use this custom port like this:
+You also have to tell meteor to use this custom port like this:
```
`docker` was correct. Meteor still uses port 80.

maxmatthews replied:
~Not if you change the meteor ENV settings, which is what the updated documentation does. If you just put port out of env, it does nothing.~ Never mind. You are right.

zodern reviewed (commented):
Thanks for fixing the documentation. There is just one thing that needs to be fixed before I merge.
